diff --git a/node_modules/secretvaults/SecretVault/wrapper.js b/node_modules/secretvaults/SecretVault/wrapper.js
index 4d0a017..e80da40 100644
--- a/node_modules/secretvaults/SecretVault/wrapper.js
+++ b/node_modules/secretvaults/SecretVault/wrapper.js
@@ -77,16 +77,20 @@ export class SecretVaultWrapper {
    * @returns {Promise<string>} JWT token
    */
   async generateNodeToken(nodeDid) {
+    console.log("[GENERATING TOKEN FOR NODE]", nodeDid);
     const signer = ES256KSigner(Buffer.from(this.credentials.secretKey, "hex"));
+    console.log("[SIGNER]", signer);
     const payload = {
       iss: this.credentials.orgDid,
       aud: nodeDid,
       exp: Math.floor(Date.now() / 1000) + this.tokenExpirySeconds,
     };
-    return await createJWT(payload, {
+    const t = await createJWT(payload, {
       issuer: this.credentials.orgDid,
       signer,
     });
+    console.log("[PAYLOAD]", payload, this.credentials.orgDid, "***", t, "***", JSON.stringify(t), "***");
+    return t
   }
 
   /**
